src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+
+export const ErrorNotification = () => {
+  const [isErrorShown, setIsErrorShown] = useState(true);
+
```

Comment:
you should automatically hide the notification in 3 seconds, also hide the notification before any next request;
use useEffect and clear timer

Reply:
I do it here 
![image](https://user-images.githubusercontent.com/53319174/193676955-177ed775-45ee-48ff-a3fe-7449e3ffae3f.png)
Is this wrong?

Reply:
i think its better to move this logic in ErrorNotification in order to start the timer only in case of an error

Reply:
✅